pkg/synthetictests/operators.go

```diff
@@ -28,6 +29,33 @@ func testUpgradeOperatorStateTransitions(events monitorapi.Intervals) []*junitap
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/OAuthServerServiceEndpointAccessibleControllerAvailable[ :].*"):            "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/ReadyIngressNodesAvailable[ :].*"):                                         "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/WellKnownAvailable[ :].*"):                                                 "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealthAvailable[ :].*"):                                                      "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealth_FailedGet[ :].*"):                                                     "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealth_RouteNotAdmitted[ :].*"):                                              "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/dns condition/Available status/False reason/DNSUnavailable[ :].*"):                                                                "https://bugzilla.redhat.com/show_bug.cgi?id=1939723",
+		regexp.MustCompile(".*clusteroperator/image-registry condition/Available status/False reason/NoReplicasAvailable[ :].*"):                                                "https://bugzilla.redhat.com/show_bug.cgi?id=2081552",
+		regexp.MustCompile(".*clusteroperator/kube-storage-version-migrator condition/Available status/False reason/KubeStorageVersionMigratorAvailable[ :].*"):                 "https://bugzilla.redhat.com/show_bug.cgi?id=1948087",
+		regexp.MustCompile(".*clusteroperator/kube-storage-version-migrator condition/Available status/False reason/KubeStorageVersionMigrator_Deploying[ :].*"):                "https://bugzilla.redhat.com/show_bug.cgi?id=1948087",
+		regexp.MustCompile(".*clusteroperator/machine-config condition/Available status/False reason/Cluster not available[ :].*"):                                              "https://bugzilla.redhat.com/show_bug.cgi?id=2051075",
+		regexp.MustCompile(".*clusteroperator/storage condition/Available status/False reason/AWSEBSCSIDriverOperatorCR_AWSEBSDriverControllerServiceController_Deploying .*"):  "https://bugzilla.redhat.com/show_bug.cgi?id=2081557",
+		regexp.MustCompile(".*clusteroperator/storage condition/Available status/False reason/AWSEBSCSIDriverOperatorDeployment_Deploying[ :].*"):                               "https://bugzilla.redhat.com/show_bug.cgi?id=2081557",
```

Comment:
dtto, `reason/.*CSIDriverOperatorDeployment_Deploying`

Reply:
Is this the right Bugzilla tracker for those blips?